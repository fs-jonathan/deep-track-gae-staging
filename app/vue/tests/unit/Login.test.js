import { shallowMount } from '@vue/test-utils'
import Login from '@/components/Login.vue'

describe('Login.vue', () => {
  it('renders correctly', () => {
    const msg = 'new message'
    const wrapper = shallowMount(Login, {
      propsData: { result: {
        msg: msg
      } }
    })
    expect(wrapper.isVueInstance()).toBeTruthy()
  })
})
