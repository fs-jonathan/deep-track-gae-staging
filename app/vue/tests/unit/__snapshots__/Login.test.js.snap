// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login.vue renders correctly 1`] = `
<div
  class="login"
  result="[object Object]"
>
  <h1>
    
  </h1>
</div>
`;
